packages/apps/tools/src/services/cross-chain-transfer-finish/get-transfer-data.ts

```diff
@@ -111,12 +107,17 @@ export async function getTransferData({
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       found?.continuation?.continuation.args as Array<any>;
 
+    const data =
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      found?.continuation?.yield as any;
+
     const { step, stepHasRollback, pactId } = found?.continuation as IPactExec;
 
+    // @ts-ignore
```

Comment:
What is TS complaining about?

Reply:
I'd prefer this to be replaced with a `@ts-expect-error <explanation>` indeed